src/app/components/accordion/accordion.ts

```diff
@@ -315,6 +313,10 @@ export class Accordion implements BlockableUI, AfterContentInit, OnDestroy {
             this.tabListSubscription.unsubscribe();
         }
     }
+
+    private isMultiple(selected: number | number[]): selected is number[] {
```

Comment:
Hi, is this type guard or what? I can't understand what this argument is for.

Reply:
Yes, it is a type guard. With the actual implementation of the accordion the _activeIndex (or tabs selected) can be a number or number[] depending of the value of the multiple property. With this type guard typescript detects correctly the typing for the argument passed (the activeIndex).
Ideally typescript could detect conditionally the type of the activeIndex property depending of the multiple value in compiling time, but this not seems to be possible for now.